src/golang/lib/collections/tests/workflow_dag_result_test.go

```diff
@@ -12,15 +13,25 @@ import (
 
 // seedWorkflowDagResultWithDags populates the workflow_dag_result table with count
 // workflow dag results where workflow_dag_id is set to the values provided in workflowDagIds.
-func seedWorkflowDagResultWithDags(t *testing.T, count int, workflowDagIds []uuid.UUID) []workflow_dag_result.WorkflowDagResult {
+func seedWorkflowDagResultWithDags(
+	t *testing.T,
+	count int,
+	now time.Time,
```

Comment:
Why does the timestamp need to be passed in here? The `seed` call should be used when you just need to create `workflowDagResults` as easily as possible. Lets remove any arguments that aren't needed.

For testing a specific columns value we should explicitly create the record.